readme.md

```diff
@@ -1,22 +1,25 @@
-# Checkpoint 05: Rails
+# Checkpoint 05: MVC & Rails
 
-#### Fork and clone this repo.
+## Set Up
 
-## Part 1 (Initial Setup)
+  Fork then clone this repository. Create commits after completing each step.
 
-1. Run the command to generate a new Rails application called `todo_app`.
-1. Create your database.  
-1. Define a `Todo` model and create a corresponding migration. It should have two attributes: `content` (string) and `is_completed` (boolean).
-1. Generate your schema.
-1. Create two Todo instances in your seed file. Run your seed file.
-1. Run your Rails server to see your application in the browser.
+## Section I (Models & Migrations)
 
-## Part 2 (Error Driven Development)
+  > Commit once after completing each step.
 
-1. Set up routes for `index`, `show` and `new`.
-1. Implement controller actions and views for the `index`, `show` and `new` features.
+  1. From the command line, generate a new Rails application called `todo_app` that uses a PostgreSQL database.
+  1. From the command line, create your database then generate a `Todo` model. `Todo` should have two attributes: `content` (string) and `is_completed` (boolean).
```

Comment:
Do we want to restrict them to using the command line to do this? Some people might want to create the files manually.